core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -100,6 +100,31 @@ message SignedKV {
   bytes previous = 3;
 }
 
+// EpochInfo contains epoch-related information that are returned in the response
+// of GetMutation APIs.
+message EpochInfo {
+  // epoch specifies the epoch of the included mutations.
+  int64 epoch = 1;
+  // smr contains the signed map root for the sparse Merkle Tree.
+  trillian.SignedMapRoot smr = 2;
+  // smh_sct is the signed certificate timestamp (SCT) showing
+  // that SMH was submitted to Trillian log.
+  // TODO: replace smh_sct with Trillian.SignedLogRoot.
+  bytes smh_sct = 3;
```

Comment:
`smh_sct` is now removed.

Thinking about inclusion in the log from a monitor perspective:
- Monitors will be comparing SMR to the leaves in the log. 
- Monitors will have some log root that they trust. 
- Inclusion proofs will need to roll up to that root. 
See #572